mimesniff/mime-types/parsing.any.js

```diff
@@ -35,7 +38,10 @@ function runTests(tests) {
 
     promise_test(() => {
       const compatibleNess = isByteCompatible(val.input);
-      if(compatibleNess === "incompatible" || compatibleNess === "header-value-incompatible") {
+
+      if(compatibleNess === "header-value-incompatible") {
+        return Promise.resolve();
```

Comment:
I recommend moving this outside the `promise_test()` wrapper and just returning early there?

Reply:
Done. Test count went down to 1898 from 1911.